Docs 10708 3.9 devkit important (#36)
mulesoft/docs-connector-devkit: modules/ROOT/pages/setting-up-your-dev-environment.adoc

endif::[]
:keywords: devkit, setup, studio, maven, devkit plugin, plugin, com.sun.tools

include::partial$devkit-important.adoc[]

> This topic explains how to prepare a development environment that includes all the components required to develop Anypoint Connectors.

Reviewer: ```suggestion
Before developing Anypoint Connectors, you must prepare a development environment that includes all the required components.
```
